releases/release-8.5.1.md

```diff
@@ -0,0 +1,102 @@
+---
+title: TiDB 8.5.1 Release Notes
+summary: Learn about the improvements and bug fixes in TiDB 8.5.1.
+---
+
+# TiDB 8.5.1 Release Notes
+
+Release date: January xx, 2025
+
+TiDB version: 8.5.1
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v8.1/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v8.1/production-deployment-using-tiup)
+
+## Operating system and platform requirement changes
+
+Starting from v8.5.1, TiDB resumes the support for CentOS Linux 7. If you plan to deploy TiDB v8.5 or upgrade your cluster to v8.5, ensure you use TiDB v8.5.1 or a later version.
+
+- According to [CentOS Linux EOL](https://www.centos.org/centos-linux-eol/), the upstream support for CentOS Linux 7 ends on June 30, 2024. In v8.4.0 DMR and v8.5.0, TiDB temporarily suspends the support for CentOS 7 and recommends you to use Rocky Linux 9.1 or later. Upgrading a TiDB cluster on CentOS 7 to v8.4.0 or v8.5.0 will cause the risk of cluster unavailability.
+
+- Although TiDB v8.5.1 and later versions resume the support for CentOS Linux 7, due to the CentOS Linux EOL status, it is strongly recommended that you review the [official announcements and security guidance](https://www.redhat.com/en/blog/centos-linux-has-reached-its-end-life-eol) for CentOS Linux 7 and consider migrating to an [operating system supported by TiDB](/hardware-and-software-requirements.md#os-and-platform-requirements), such as Rocky Linux 9.1 or later.
+
+## Improvements
+
++ TiDB <!--tw@Oreoxmt: 5 notes-->
+
+    - Support folding read-only user-defined variables into constants [#52742](https://github.com/pingcap/tidb/issues/52742) @[winoros](https://github.com/winoros)
+    - Convert Cartesian product Semi Join with nulleq condition to Semi Join with equality condition [#57583](https://github.com/pingcap/tidb/issues/57583) @[hawkingrei](https://github.com/hawkingrei)
+    - Adjust the default threshold of statistics memory cache to 20% of total memory [#58014](https://github.com/pingcap/tidb/issues/58014) @[hawkingrei](https://github.com/hawkingrei)
+
++ TiKV <!--tw@Oreoxmt: 1 note-->
+
+    - Add detection mechanism for illegal `max_ts` updates [#17916](https://github.com/tikv/tikv/issues/17916) @[ekexium](https://github.com/ekexium)
+
++ TiFlash
+
+    - (dup): release-7.5.5.md > Improvements> TiFlash - Optimize the retry strategy for TiFlash compute nodes in the disaggregated storage and compute architecture to handle exceptions when downloading files from Amazon S3 [#9695](https://github.com/pingcap/tiflash/issues/9695) @[JinheLin](https://github.com/JinheLin)
+
++ Tools
+
+    + TiCDC <!--tw@qiancai: 1 note-->
+
+        - 减少 TiCDC 在做 Initial Scan 时对与 TiKV 中 cache hit rate 的影响 [#17877](https://github.com/tikv/tikv/issues/17877) @[hicqu](https://github.com/hicqu)
```

Comment:
```suggestion
        - Filter out events that are not subscribed to by TiCDC in advance to avoid unnecessary resource consumption [#17877](https://github.com/tikv/tikv/issues/17877) @[hicqu](https://github.com/hicqu)
```